### PR TITLE
dra: init at 2022-08-13

### PR DESCRIPTION
**Pull request by Mertzenich**
###### Description of changes

Add package (https://github.com/eastrocky/dra) which provides the Douay-Rheims 1899 American Edition Bible on the command line. Similar to [vul](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/vul/default.nix).

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [X] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by ocfox — CHANGES_REQUESTED** [review]
Where is the top-level entry?

**Comment by Mertzenich** [comment]
> Where is the top-level entry?

Forgot to add the change, should be rectified. Please let me know if there is anything else I mised.

File: pkgs/applications/misc/dra/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "dra";
+  version = "2022-08-13";
```
Comment (ocfox) [inline pkgs/applications/misc/dra/default.nix:5]:
```suggestion
  version = "unstable-2022-08-13";
```
See https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming

**Review by ocfox — CHANGES_REQUESTED** [review]
And squash to two commits.

**Comment by Mertzenich** [comment]
> And squash to two commits.

Should be done, I believe I acted appropriately.

**Comment by ocfox** [comment]
But this a fork of https://github.com/LukeSmithxyz/vul, and is it really a useful package?

**Comment by Mertzenich** [comment]
> But this a fork of https://github.com/LukeSmithxyz/vul, and is it really a useful package?

Its providing a different translation so I would say it's useful, however upon second look I have [found a fork](https://github.com/thenewmantis/bbl) which supports multiple translations which seems like it will be more useful (provides all of the features of `vul` this package, and more. I think I'll close this and package that instead.

**Pull request closed without merging**